server/tests/meter/test_filter.py

```diff
@@ -26,6 +27,59 @@ def test_strip_metadata_prefix() -> None:
     assert clause.property == "property"
 
 
+class TestFilterClauseValueValidation:
+    """Test validation of FilterClause.value field to prevent database overflow."""
+
+    def test_valid_integer_values(self) -> None:
+        """Test that valid integers within int32 range are accepted."""
+        # Test boundary values
+        max_valid = FilterClause(property="test", operator=FilterOperator.eq, value=2147483647)
+        assert max_valid.value == 2147483647
+        
+        min_valid = FilterClause(property="test", operator=FilterOperator.eq, value=-2147483648)
+        assert min_valid.value == -2147483648
+        
+        # Test normal value
+        normal = FilterClause(property="test", operator=FilterOperator.eq, value=123)
+        assert normal.value == 123
+        
+        # Test zero
+        zero = FilterClause(property="test", operator=FilterOperator.eq, value=0)
+        assert zero.value == 0
+
+    def test_invalid_integer_values(self) -> None:
```

Comment:
Please use `@pytest.mark.parametrize` to generate those different cases. Docs: https://docs.pytest.org/en/6.2.x/parametrize.html